qutip/solver.py

```diff
@@ -79,9 +90,12 @@ def step(self, iter_, state):
         else:
             t = self.tlist[iter_]
             for ii in range(self.e_num):
-                self.raw_out[ii, iter_] = \
-                    self.e_ops_qoevo[ii].compiled_qobjevo.expect(t, state)
-
+                if isinstance(ii, (Qobj, QobjEvo, int)):
```

Comment:
I missed that! Updated.

Reply:
This test is still used in 4 others places.

Reply:
I noticed I was testing if `callable(ii)` which was clearly incorrect, and just updated it. Otherwise, I am not sure what you mean by saying this test is still used in 4 other places. Could you clarify? I double checked the diff and I no longer see testing if `ii` is of type `int`.